staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_filter.go

```diff
@@ -82,15 +84,17 @@ func NewTestable(
 	flowcontrolClient fcclientv1a1.FlowcontrolV1alpha1Interface,
 	serverConcurrencyLimit int,
 	requestWaitLimit time.Duration,
+	obsPairGenerator metrics.TimedObserverPairGenerator,
 	queueSetFactory fq.QueueSetFactory,
 ) Interface {
-	return newTestableController(informerFactory, flowcontrolClient, serverConcurrencyLimit, requestWaitLimit, queueSetFactory)
+	return newTestableController(informerFactory, flowcontrolClient, serverConcurrencyLimit, requestWaitLimit, obsPairGenerator, queueSetFactory)
 }
 
 func (cfgCtl *configController) Handle(ctx context.Context, requestDigest RequestDigest,
```

Comment:
ctl is super confusing here because it is the abbreviation for `control` (like kubectl or sysctl). For controller, it would probably be clearer to use `ctlr` (like we do in client-go, https://github.com/kubernetes/kubernetes/blob/47f5d2923f3f35adc66b4797a95720f67c948b4e/staging/src/k8s.io/client-go/tools/cache/controller.go#L114)

Reply:
changed.